web/webpack.plugins.js

```diff
@@ -1,7 +1,6 @@
 const webpack = require('webpack');
 const MinifyPlugin = require('babel-minify-webpack-plugin');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
-const Dotenv = require('dotenv-webpack');
 
 // Define webpack plugins used in dev and prod configurations here
```

Comment:
Can you explain why Dotenv was moved from `webpack.plugins.js` to `webpack.config.js`?

Reply:
To keep dev config hidden - builds use `webpack.plugins.js` and `webpack.prod.js`, the dev-server uses `webpack.config.js`. This prevents us from leaking credentials in .env into build bundbles

Reply:
Got it :ok_hand: In that case we should probably update the comment on line 5 of `webpack.plugins.js`. Could also make it extra clear by renaming `webpack.config.js` to `webpack.dev.js`.

Reply:
doesn't line 5 already say something along those lines?

```
// Define webpack plugins used in dev and prod configurations here
```

Reply:
Oh my bad, is that file used for both the prod build and the dev server? If so, no changes needed